node-rcon.js

```diff
@@ -71,14 +83,25 @@ Rcon.prototype.send = function(data, cmd, id) {
   this._sendSocket(sendBuf);
 };
 
+/**
+ * Sends and establishes a socket connection with the server, either on TCP or UDP connections
+ * @param {*} buf - The buffer inherited from the connection
+ */
 Rcon.prototype._sendSocket = function(buf) {
-  if (this._tcpSocket) {
-    this._tcpSocket.write(buf.toString('binary'), 'binary');
-  } else if (this._udpSocket) {
-    this._udpSocket.send(buf, 0, buf.length, this.port, this.host);
-  }
+  if (this._udpSocket)
+    return this._udpSocket.send(buf, 0, buf.length, this.port, this.host);
+  this._tcpSocket.write(buf.toString('binary'), 'binary');
```

Comment:
What was the reason for the changes in this function?

Reply:
There wasn't a major reason to change this other than the coding style and readability, and, on a technical side note, to reduce redundancy. Tell me if it's good or it should be changed, I'm open to hear